### PR TITLE
Folder structure

### PR DESCRIPTION
**Pull request by Prakash-kumar1**

### CODE REVIEW

File: src/components/Donut/Donutchart.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from "react";
+import Chart from "react-apexcharts";
+import { ToastContainer, toast } from 'react-toastify';
```
Comment (Souvikdeb2612) [inline src/components/Donut/Donutchart.js]: format imports properly

File: src/components/Donut/Donutchart.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      .then((response) => {
+        setMydata(response.data);
+      })
+      .catch(()=>{
+        {notify()} }) 
+  }, []);
+
+  const country = [];
+  const medal = [];
+  mydata.map((value) => {
```
Comment (Souvikdeb2612) [inline src/components/Donut/Donutchart.js:28]: destructure value here

File: src/components/Line/Linechart.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from "react";
+import Chart from "react-apexcharts";
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
```
Comment (Souvikdeb2612) [inline src/components/Line/Linechart.js:5]: format imports here

File: src/components/Main/Main.js [hunk 4]
```diff
@@ -9,11 +9,12 @@ import Covid from "../Covid/Covid";
...
 import Calender from "../calender/Calender";
 import Footer from "../Footer/Footer"
-import RichestPerson from "../RichestPerson/RichestPerson"
-import Population from "../Population/Population"
-import WorldMap from "../Worldmap/WorldMap"
-import Converter from "../ExchangeRate/Converter"
-
+import {Donutchart} from "../Donut/Donutchart" ;
+import {Linechart} from "../Line/Linechart" ;
+import { Quotes } from "../Quote/Quotes";
```
Comment (Souvikdeb2612) [inline src/components/Main/Main.js:14]: format imports

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/components/Donut/Donutchart.js (hunk 1 above)
Comment (Prakash-kumar1) [reply src/components/Donut/Donutchart.js]: Sir , this imports from external libraries

File: src/components/Line/Linechart.js (hunk 3 above)
Comment (Prakash-kumar1) [reply src/components/Line/Linechart.js:5]: Sir , this imports from external libraries

File: src/components/Main/Main.js (hunk 4 above)
Comment (Prakash-kumar1) [reply src/components/Main/Main.js:14]: Sir , this imports from external libraries

File: src/components/Donut/Donutchart.js (hunk 2 above)
Comment (Prakash-kumar1) [reply src/components/Donut/Donutchart.js:28]: Sir , i have used map outside return . so, there is no need to destructuring the value

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**